zkbind/src/org/zkoss/bind/impl/ParamCall.java

```diff
@@ -232,6 +233,9 @@ public Object resolveParameter(Annotation anno, Class<?> returnType) {
 						} else {
 							throw new UiException("don't use " + s + " with other scopes " + Arrays.toString(ss));
 						}
+						if (val == null) {
+							val = Executions.getCurrent().getAttribute(name);
+						}
```

Comment:
I think this block is unnecessary. The `Components.getImplicit` would be called below.